docs/articles/documentation/reference/api/testcafe/createbrowserconnection.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: docs
+title: TestCafe.createBrowserConnection Method
+permalink: /documentation/reference/api/testcafe/createbrowserconnection.html
+---
+# TestCafe.createBrowserConnection Method
+
+Creates a [remote browser connection](../browserconnection/README.md).
+
+```text
+async createBrowserConnection() → Promise<BrowserConnection>
+```
+
+To connect a remote browser, navigate it to [BrowserConnection.url](../browserconnection/url.md).
```

Comment:
"navigate it to" What does 'it' refer to here?

Reply:
I thought you can "navigate a browser to a URL". So, it should probably be just:

*To connect a remote browser, navigate to BrowserConnection.url.*